packages/vuetify/src/util/helpers.ts

```diff
@@ -150,6 +150,33 @@ export function addOnceEventListener (el: EventTarget, event: string, cb: () =>
   el.addEventListener(event, once, false)
 }
 
+export function addOptionsSafeEventListener (
+  el: EventTarget,
+  event: string,
+  cb: EventHandlerNonNull | (() => void),
+  options: {}
+): void {
+  const optionsSupported = Object.keys(options).reduce((obj, key) => {
+    obj[key] = false
+    return obj
+  }, Object())
+
+  try {
+    // IE does not support passing an options object
+    const testOpts = Object.keys(options).reduce((obj, key) => {
+      Object.defineProperty(obj, key, {
+        get: () => {
+          optionsSupported[key] = true
+        }
+      })
+    }, Object())
+    el.addEventListener('testListener', () => {}, testOpts)
+    el.removeEventListener('testListener', () => {}, testOpts)
+  } catch (e) { console.log(e) }
```

Comment:
This should be rather `console.warn()` (`console.error()`?) but I'm not sure if this is even required, maybe you could skip it.